enterprise_subsidy/apps/api/v1/views/transaction.py

```diff
@@ -452,7 +453,7 @@ def create(self, request, *args, **kwargs):
                 status=status.HTTP_400_BAD_REQUEST,
             )
         try:
-            transaction, created = subsidy.redeem(learner_id, content_key, subsidy_access_policy_uuid)
+            transaction, created = subsidy.redeem(learner_id, content_key, subsidy_access_policy_uuid, metadata)
```

Comment:
We should have a test to cover this.  I think this call is actually causing the value of `metadata` to be passed through the `idempotency_key` argument, because args/kwargs.